src/main/java/org/opensrp/domain/postgres/IdentifierSource.java

```diff
@@ -0,0 +1,366 @@
+package org.opensrp.domain.postgres;
+
+public class IdentifierSource {
```

Comment:
We need a **regex_format** field here that is optional. This will be used to define the pattern of numbers that can be generated